provider/examples/resources/sec/sec/main.tf

```diff
@@ -0,0 +1,29 @@
+data "aws_route53_zone" "selected" {
```

Comment:
The example structure seems really off here. Can you put the sec directory inside modules?

Reply:
I will take a note, to fix it properly will need to fix other examples as well, so I will leave it for now and fix them all together when we want to provide complete examples.